src/arvgcboolean.c

```diff
@@ -213,6 +213,27 @@ arv_gc_boolean_set_value (ArvGcBoolean *gc_boolean, gboolean v_boolean, GError *
 		g_propagate_error (error, local_error);
 }
 
+static ArvGcAccessMode
+arv_gc_boolean_get_access_mode (ArvGcFeatureNode *gc_feature_node)
```

Comment:
The content of this function is duplicated in several instances. It looks like the only difference is the access to pValue. It is a call for the creation of a get_pointed_node virtual in ArvGcFeature. May be we don't need an access_mode virtual after all.